File: trunk/src/Client/communications/ServerProxy.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using IServer;

namespace Client.communications
{
   public class ServerProxy{
       private static ServerPrx server;


       public static void setServerProxy(ServerPrx serverPrx){
           server = serverPrx;
       }


       public static ServerPrx getInstance(){
           return server;
       }

       
   }
}

File: trunk/src/Client/src/communications/ServerProxy.java
package communications;

import IServer.ServerPrx;

public class ServerProxy {
	private static ServerPrx server;
	
	public static void setServerProxy(ServerPrx server){
		ServerProxy.server = server;
	}
	
	public static ServerPrx getProxy(){
		return server;
	}
}
